core/src/main/java/com/linecorp/armeria/common/logging/DefaultRequestLog.java

```diff
@@ -538,16 +545,21 @@ private void startResponse0(long responseStartTimeNanos, long responseStartTimeM
         }
 
         this.responseStartTimeNanos = responseStartTimeNanos;
-        this.responseStartTimeMillis = responseStartTimeMillis;
+        this.responseStartTimeMicros = responseStartTimeMillis;
```

Comment:
Could you change the name of the parameter(`responseStartTimeMillis`) as well?

Reply:
Oops thanks